examples/microprofile/telemetry/greeting/README.md

```diff
@@ -0,0 +1,59 @@
+# Helidon MicroProfile Telemetry Example
+
+This example implements demonstrates usage of MP Telemetry Tracing.
+
+## Build and run
+
+With JDK17+
+```bash
+mvn package
+java -jar target/helidon-examples-microprofile-telemetry-greeting.jar
```

Comment:
The README file is at the top-level directory of this multi-module project. If the user runs `mvn package` from there as suggested (which is reasonable) -- and note that, as written, there are no instructions to change the directory -- then the first `java` command does not work because it does not include the intervening directory nor are there instructions to change to the `greeting` directory.

Perhaps have the user stay at the top-level directory throughout these steps, and change the two `java` commands to:
```
java -jar greeting/target/helidon-examples-microprofile-telemetry-greeting.jar
```

and 
```
java -jar secondary/target/helidon-examples-microprofile-telemetry-secondary.jar
```
(and remove the instruction to change to the `secondary` subdirectory before running the second `java` command).

Reply:
Thanks for the suggestion! Applied!